scripts/get_qemu.sh

```diff
@@ -1,30 +1,35 @@
 #!/usr/bin/env bash
+# Download and make qemu-user-static [1] executable in QEMU_DIR
+# for various architectures (archs)
+# [1]: https://github.com/multiarch/qemu-user-static/
+set -e
 
 CURRENT_DIR=$(pwd)
-QEMU_STATIC_VERSION=v3.1.0-3
 QEMU_DIR="./build/qemu/"
+# qemu-user-static version and download URL
+QEMU_STATIC_VERSION=v5.1.0-7
+QEMU_REL_URL="https://github.com/multiarch/qemu-user-static/releases/download"
 
-qemu_ppc64le_sha256=d018b96e20f7aefbc50e6ba93b6cabfd53490cdf1c88b02e7d66716fa09a7a17
-qemu_aarch64_sha256=a64b39b8ce16e2285cb130bcba7143e6ad2fe19935401f01c38325febe64104b
-qemu_arm_sha256=f4184c927f78d23d199056c5b0b6d75855e298410571d65582face3159117901
-qemu_x86_64_sha256=b9e444bf656c13a6db502f09e3135ef0c6045a117d5413662ba233d8b80f8fbd
+# Supported archs list
+declare -a archs=("ppc64le" "aarch64" "arm" "x86_64")
+
+# SHA256 of each architecture / archive: qemu-${arch}-static
+ppc64le=c850cb81287f9be43be181a6513803b03da5e1b76b9ab6d727566f1651889b37
+aarch64=06664613db8785b600412d63b572f31edeafe0c06b655d51d16a536d27cdf47b
+arm=977cd7e1a1cdcf53b3873f2348423fa29de99db5fa2ee77b4491039ae7e0600a
+x86_64=d753aa1904531e251443b887e47b7e415fb8a99334225911250bef4c8ae4bc4e
 
 mkdir -p $QEMU_DIR/
 cd $QEMU_DIR/
 
-wget https://github.com/multiarch/qemu-user-static/releases/download/${QEMU_STATIC_VERSION}/qemu-ppc64le-static
-wget https://github.com/multiarch/qemu-user-static/releases/download/${QEMU_STATIC_VERSION}/qemu-aarch64-static
-wget https://github.com/multiarch/qemu-user-static/releases/download/${QEMU_STATIC_VERSION}/qemu-arm-static
-wget https://github.com/multiarch/qemu-user-static/releases/download/${QEMU_STATIC_VERSION}/qemu-x86_64-static
-
-sha256sum qemu-ppc64le-static | grep -F "${qemu_ppc64le_sha256}"
-sha256sum qemu-aarch64-static | grep -F "${qemu_aarch64_sha256}"
-sha256sum qemu-arm-static | grep -F "${qemu_aarch64_sha256}"
-sha256sum qemu-x86_64-static | grep -F "${qemu_x86_64_sha256}"
-
-chmod +x qemu-ppc64le-static
-chmod +x qemu-aarch64-static
-chmod +x qemu-arm-static
-chmod +x qemu-x86_64-static
+for arch in "${archs[@]}"; do 
```

Comment:
For loops are someetimes more confusing than copy paste.

Maybe i should just learn more bash.